sdk/go/common/workspace/paths_test.go

```diff
@@ -90,7 +90,7 @@ func TestProjectStackPath(t *testing.T) {
 		"name: some_project\ndescription: Some project\nruntime: nodejs\nconfig: stacksA\nstackConfigDir: stacksB\n",
 		func(t *testing.T, projectDir, path string, err error) {
 			assert.Error(t, err)
-			assert.Equal(t, "can not set `config` and `stackConfigDir`, remove the `config` entry", err.Error())
```

Comment:
I think the original wording was better here, makes it clear that "stackConfigDir" is the one to keep.

Reply:
Again I still prefer the original wording here telling the user to use "stackConfigDir"

Reply:
The assert uses `Contains` to partially check the error message. The full error message is this:
> Should not use both config and stackConfigDir to define the stack directory. Use only stackConfigDir instead.

Reply:
See no reason for the test to not assert that then.